pages/common/bower.md

```diff
@@ -0,0 +1,32 @@
+# bower
+
+> A package manager optimized for front-end web development.
+> A package can be a GitHub shorthand, a Git endpoint, a URL or a registered package.
+
+- Install the project dependencies listed in bower.json:
+
+`bower install`
+
+- Install packages to the bower_components directory:
+
+`bower install {{endpoint}} {{endpoint}}`
```

Comment:
"endpoint" is slightly unclear here. Any chance we could use a different token name?